### PR TITLE
repl cflags: plumbing

### PR DESCRIPTION
**Pull request by amosr**
tedious
/jury approved @jystic

### CODE REVIEW

File: icicle-compiler/src/Icicle/Repl.hs [hunk 1]
```diff
@@ -93,6 +93,8 @@ getInitialState = do
                 []
               UseColor ->
                 [FlagColor]
+
+      , stateCFlags = "-O0 -UICICLE_ASSERT"
```
Comment (jystic) [inline icicle-compiler/src/Icicle/Repl.hs]: Could this be `-DICICLE_ASSERT=0` instead?

**Comment by jystic** [comment]
👍 looks good

File: icicle-compiler/src/Icicle/Repl.hs (hunk 1 above)
Comment (amosr) [reply icicle-compiler/src/Icicle/Repl.hs]: nah, gcc 4.7 barfs if you pass the same identifier multiple times as arguments like "-DA=1 -DA=2"

Comment (jystic) [reply icicle-compiler/src/Icicle/Repl.hs]: wow, that's delightful

Comment (amosr) [reply icicle-compiler/src/Icicle/Repl.hs]: real helpful

**Pull request merged**